api-examples/transformer_utils.py

```diff
@@ -407,3 +458,29 @@ def forward(self, inputs, targets):
                 pred = self.model(in_)
                 return self._loss(pred, targets)
         return LossFn(self, loss)
+
+
+def create_model(embeddings, d_model, d_ff, dropout, num_heads, num_layers, model_type, rpr_k, d_k, reduction_d_k,
```

Comment:
create_paired_model?

Reply:
this also create seq2seq model, so i'd like to keep it

Reply:
seq2seq is still paired.  the problem is the current name is not specific